node-operators/networks/compile-binary.md

```diff
@@ -30,24 +30,25 @@ Let's check out the latest release:
 git checkout tags/$(git tag | tail -1)
 ```
 
-Next, install a Substrate development environment, including Rust, by executing:
+Next, install Rust and its prerequisites [via Rust's recommended method](https://www.rust-lang.org/tools/install){target=_blank} by executing:
 
 ```
---8<-- 'code/setting-up-node/substrate.md'
+--8<-- 'code/setting-up-node/installrust.md'
 ```
 
-Lastly, build parachain binary:
+Next, update your PATH environment variable by running:
```

Comment:
This is only necessary if they just installed Rust, if you have rust install you can skip some steps. You might as well include that information somewhere like "If you already have Rust installed, go ahead and skip ....." or something